main.go

```diff
@@ -89,33 +104,41 @@ func main() {
 
 	parseFlags()
 	testNetwork := network.New(
-		network.Nodes(config.NodesCount, multiverse.NewNode, network.ZIPFDistribution(config.ZipfParameter, float64(config.NodesTotalWeight))),
-		network.Delay(30*time.Millisecond, 250*time.Millisecond),
-		network.PacketLoss(0, 0.05),
+		network.Nodes(config.NodesCount, multiverse.NewNode, network.ZIPFDistribution(
+			config.ZipfParameter, float64(config.NodesTotalWeight))),
+		network.Delay(time.Duration(config.DecelerationFactor)*time.Duration(config.MinDelay)*time.Millisecond,
+			time.Duration(config.DecelerationFactor)*time.Duration(config.MaxDelay)*time.Millisecond),
+		network.PacketLoss(0, config.PayloadLoss),
 		network.Topology(network.WattsStrogatz(4, 1)),
 	)
 	testNetwork.Start()
 	defer testNetwork.Shutdown()
 
-	awResultsWriters := monitorNetworkState(testNetwork)
-	defer flushWriters(awResultsWriters)
+	resultsWriters := monitorNetworkState(testNetwork)
+	defer flushWriters(resultsWriters)
 	secureNetwork(testNetwork, config.DecelerationFactor)
 
-	time.Sleep(2 * time.Second)
+	// time.Sleep(2 * time.Second)
+
+	// To simulate the confirmation time w/o any double spendings, the colored msgs are not to be sent
+
+	// Here we simulate the double spending
+	// sendMessage(testNetwork.Peers[0], multiverse.Blue)
+	// sendMessage(testNetwork.Peers[0], multiverse.Red)
```

Comment:
Do we want to have this permenently commented out? Or maybe should this be controlled via some parameter?

Reply:
We can define some parameters to control this. So user can define currently which simulation they want to run 1) confirmation time tracing 2) double spending tracing.